src/server/models/tests/mediaquery.test.js

```diff
@@ -0,0 +1,121 @@
+require('dotenv').config();
+require('babel-polyfill');
+const client = require('../db');
+const { getCastsByTerm } = require('../mediaquery');
+
+const dbURL = `${process.env.DB_URL}${process.env.DB_DATABASE}?replicaSet=${process.env.DB_REPLSET}`;
+const testCollectionName = 'mediaquerytests';
+
+beforeAll(async done => {
+  client.connect(dbURL, function() {});
```

Comment:
@chern One of the suggestions to getting rid of the async errors is to pass in

jest.setTimeout(10000);

See https://stackoverflow.com/a/52543554/6001

Reply:
Hmm, in the latest commit I added `jest.setTimeout(30000);`, but it still encounters the async error